mlos_bench/mlos_bench/config/services/remote/azure/scripts/setup-rg/rg-template.json

```diff
@@ -0,0 +1,346 @@
+            }
+        }
+    },
+    {
+        "name": "[variables('kvName')]",
+        "type": "Microsoft.KeyVault/vaults",
+        "apiVersion": "2019-09-01",
+        "location": "[parameters('vmLocation')]",
+        "dependsOn": [
+            "[resourceId('Microsoft.Network/virtualNetworks', variables('vnetName'))]"
+        ],
+        "tags": {
+            "displayName": "[variables('kvName')]"
+        },
+        "properties": {
+            "enabledForDeployment": true,
+            "enabledForTemplateDeployment": true,
+            "enabledForDiskEncryption": true,
+            "tenantId": "72f988bf-86f1-41af-91ab-2d7cd011db47",
+            "accessPolicies": [],
+            "enableRbacAuthorization": true,
+            "networkAcls": {
+                "virtualNetworkRules": [
+                    {
+                        "id": "[resourceId('Microsoft.Network/virtualNetworks/subnets', variables('vnetName'), 'default')]",
+                        "ignoreMissingVnetServiceEndpoint": false
+                    }
+                ]
+            },
+            "sku": {
+                "name": "standard",
+                "family": "A"
+            }
+        }
+    },
+    {
+        "name": "[variables('vmName')]",
+        "type": "Microsoft.Compute/virtualMachines",
+        "apiVersion": "2021-03-01",
+        "location": "[parameters('vmLocation')]",
+        "dependsOn": [
+            "[resourceId('Microsoft.Network/networkInterfaces', variables('networkInterfaceName'))]"
+        ],
+        "tags": {
+            "displayName": "[variables('vmName')]"
+        },
+        "properties": {
+            "hardwareProfile": {
+                "vmSize": "[parameters('vmSKU')]"
+            },
+            "osProfile": {
+                "computerName": "[variables('vmName')]",
+                "adminUsername": "[parameters('vmAdminUsername')]",
+                "linuxConfiguration": {
+                    "disablePasswordAuthentication": true,
+                    "ssh": {
+                        "copy": [
+                            {
+                                "name": "publicKeys",
+                                "count": "[length(parameters('sshPublicKeys'))]",
+                                "input": {
+                                    "path": "[concat('/home/', parameters('vmAdminUsername'), '/.ssh/authorized_keys')]",
+                                    "keyData": "[parameters('sshPublicKeys')[copyIndex('publicKeys')]]"
+                                }
+                            }
+                        ]
+                    }
+                }
+            },
+            "storageProfile": {
+                "imageReference": {
+                    "publisher": "Canonical",
+                    "offer": "0001-com-ubuntu-server-jammy",
+                    "sku": "22_04-lts-gen2",
+                    "version": "latest"
```

Comment:
Any way to remove the version from this entirely and just let it take "Ubuntu Server LTS Latest"?

Reply:
There doesn't seem to be a simple way to do this without resorting to external querying like:
```sh
> az vm image list --all --publisher Canonical \
  | jq '[.[] | select(.sku | endswith("lts-gen2"))] | max_by(.version)'

{
  "architecture": "x64",
  "offer": "0001-com-ubuntu-server-jammy-daily",
  "publisher": "Canonical",
  "sku": "22_04-daily-lts-gen2",
  "urn": "Canonical:0001-com-ubuntu-server-jammy-daily:22_04-daily-lts-gen2:22.04.202308280",
  "version": "22.04.202308280"
}
```
which would complicate the script if trying to use as a ARM parameter. What do you think about this sort of querying?

Reply:
LGTM, thanks!

Reply:
Tho, let's just leave a note about this in the comments for now and leave it for a future PR.

Reply:
Added a TODO with this!